### PR TITLE
Update idebugcodecontext2-getdocumentcontext.md

### PR DESCRIPTION
**Pull request by pieandcakes**
@gregg-miskelly Is the statement I put clearer? Or do you have a suggestion on what we should say to make it more clear?

### CODE REVIEW

**Comment by PRMerger7** [comment]
@pieandcakes : Thanks for your contribution! The author, @madskristensen, has been notified to review your proposed change.

File: docs/extensibility/debugger/reference/idebugcodecontext2-getdocumentcontext.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ int GetDocumentContext( 
 
 ## Remarks
  Generally, the document context can be thought of as a position in a source file while the code context is a position of a code instruction in an execution stream.
+ 
+ If no document context is available, returning `S_FALSE` is the desired behavior.
```
Comment (gregg-miskelly) [inline docs/extensibility/debugger/reference/idebugcodecontext2-getdocumentcontext.md]: Doesn't this note belong under 'Return Value'?

Comment (pieandcakes) [reply docs/extensibility/debugger/reference/idebugcodecontext2-getdocumentcontext.md]: I moved this comment and reworded it a bit.

**Comment by PRMerger7** [comment]
@pieandcakes : Thanks for your contribution! The author, @madskristensen, has been notified to review your proposed change.

**Comment by madskristensen** [comment]
#sign-off

**Pull request merged**